### PR TITLE
add loginpage class

### PR DESCRIPTION
**Pull request by raulchoque-eng**
Add LoginPage Class.

### CODE REVIEW

File: src/test/java/trello/steps/ManagerSteps.java [hunk 1]
```diff
@@ -1,4 +1,22 @@
...
+ * Jala Foundation, ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jala Foundation.
+ */
+
 package trello.steps;
 
+/**
+ * Key class.
```
Comment (MelissaRoman06) [inline src/test/java/trello/steps/ManagerSteps.java]: Please change the documentation with the right class name.

File: src/test/java/trello/hooks/Hooks.java [hunk 2]
```diff
@@ -1,4 +1,22 @@
...
+ * Jala Foundation, ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jala Foundation.
+ */
+
 package trello.hooks;
 
+/**
+ * Key class.
```
Comment (MelissaRoman06) [inline src/test/java/trello/hooks/Hooks.java]: Please change the documentation with the right class name.

File: src/test/java/trello/steps/ManagerSteps.java (hunk 1 above)
Comment (raulchoque-eng) [reply src/test/java/trello/steps/ManagerSteps.java]: Thank you, this wrong was fixed.

File: src/main/java/trello/keys/NamePages.java [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+/**
+ * NamePages class.
+ *
+ * @author Raul Choque
+ * @version 0.0.1
+ */
+public class NamePages {
+
+    private User user;
+    private String baseUrl = "https://trello.com";
```
Comment (MelissaRoman06) [inline src/main/java/trello/keys/NamePages.java]: I think this could be a constant.

File: src/main/java/trello/ui/PageTransporter.java [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+import org.openqa.selenium.WebDriver;
+
+/**
+ * PageTransporter class.
+ *
+ * @author Raul Choque
+ * @version 0.0.1
+ */
+public class PageTransporter {
+
```
Comment (MelissaRoman06) [inline src/main/java/trello/ui/PageTransporter.java:25]: An unnecessary empty line.

File: src/main/java/trello/ui/pages/HomePage.java [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+/**
+ * HomePage class.
+ *
+ * @author Raul Choque
+ * @version 0.0.1
+ */
+public class HomePage extends BasePage {
+
+
```
Comment (MelissaRoman06) [inline src/main/java/trello/ui/pages/HomePage.java]: An unnecessary empty line.

File: src/main/java/trello/ui/pages/LoginPage.java [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+    }
+
+    /**
+     * Clicks to submit login form.
+     */
+    public void clickSubmit() {
+        logInBtn.click();
+    }
+
+
```
Comment (MelissaRoman06) [inline src/main/java/trello/ui/pages/LoginPage.java]: An unnecessary empty line.

File: src/test/java/trello/steps/LoginSteps.java [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+     * Constructor method for share states between objects.
+     *
+     * @param context has all share entities.
+     */
+    public LoginSteps(final Context context) {
+        this.context = context;
+        this.user = context.getUser();
+    }
+
+
```
Comment (MelissaRoman06) [inline src/test/java/trello/steps/LoginSteps.java]: An unnecessary empty line.

File: src/test/java/trello/hooks/Hooks.java (hunk 2 above)
Comment (raulchoque-eng) [reply src/test/java/trello/hooks/Hooks.java]: Thank you, this wrong was fixed.

File: src/main/java/trello/keys/NamePages.java (hunk 3 above)
Comment (raulchoque-eng) [reply src/main/java/trello/keys/NamePages.java]: Thank you, this wrong was fixed.

File: src/main/java/trello/ui/PageTransporter.java (hunk 4 above)
Comment (raulchoque-eng) [reply src/main/java/trello/ui/PageTransporter.java:25]: Thank you, this wrong was fixed.

File: src/main/java/trello/ui/pages/HomePage.java (hunk 5 above)
Comment (raulchoque-eng) [reply src/main/java/trello/ui/pages/HomePage.java]: Thank you, this wrong was fixed.

File: src/main/java/trello/ui/pages/LoginPage.java (hunk 6 above)
Comment (raulchoque-eng) [reply src/main/java/trello/ui/pages/LoginPage.java]: Thank you, this wrong was fixed.

File: src/test/java/trello/steps/LoginSteps.java (hunk 7 above)
Comment (raulchoque-eng) [reply src/test/java/trello/steps/LoginSteps.java]: Thank you, this wrong was fixed.

**Review by MelissaRoman06 — APPROVED** [review]

**Pull request merged**